Ruby 2.7 on M1 mac

First off, I have to use `sudo` because `asdf install ruby 2.7.5` begets:
```
Installing ruby-2.7.5...
mkdir: /opt/homebrew: Permission denied
```
I tried updating permissions on `/opt` but get Permission denied on that too.

So I run `sudo asdf install ruby 2.7.5` and get further, failing with this log:
```
...
compiling readline.c
linking shared-object monitor.bundle
linking shared-object pty.bundle
linking shared-object syslog.bundle
compiling objspace_dump.c
readline.c:1904:37: error: use of undeclared identifier 'username_completion_function'; did you mean 'rl_username_completion_function'?
                                    rl_username_completion_function);
                                    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                                    rl_username_completion_function
readline.c:79:42: note: expanded from macro 'rl_username_completion_function'
# define rl_username_completion_function username_completion_function
                                         ^
/usr/local/opt/readline/include/readline/readline.h:485:14: note: 'rl_username_completion_function' declared here
extern char *rl_username_completion_function PARAMS((const char *, int));
             ^
1 error generated.
make[2]: *** [readline.o] Error 1
installing default digest libraries
linking shared-object strscan.bundle
make[1]: *** [ext/readline/all] Error 2
make[1]: *** Waiting for unfinished jobs....
installing default nkf libraries
compiling date_parse.c
linking shared-object digest.bundle
compiling console.c
compiling date_strftime.c
installing default console libraries
installing digest libraries
compiling date_strptime.c
installing default date_core libraries
linking shared-object objspace.bundle
linking shared-object io/console.bundle
linking shared-object nkf.bundle
linking shared-object date_core.bundle
make: *** [build-ext] Error 2
```

I was able to install using `arch -x86_64`, but it broke all commands that use ruby, so I don't think that's right.

--- a/README.md
+++ b/README.md
@@ -27,7 +27,9 @@ RUBY_APPLY_PATCHES=$'dir/1.patch\n2.patch\nhttp://example.com/3.patch' asdf inst
 RUBY_APPLY_PATCHES=$(curl -s https://raw.githubusercontent.com/rvm/rvm/master/patchsets/ruby/2.1.1/railsexpress) asdf install ruby 2.1.1
 ```
 
-Running `asdf plugin-update ruby` will update asdf-ruby and ensure the latest versions of ruby are available to install. By default asdf-ruby uses the latest release of ruby-build, however instead you can choose your own branch/tag through the `ASDF_RUBY_BUILD_VERSION` variable:
+> **_NOTE:_**  This plugin does not automatically fetch new Ruby versions. Running `asdf plugin-update ruby` will update asdf-ruby and ensure the latest versions of Ruby are available to install.
+
+By default asdf-ruby uses a recent release of ruby-build, however instead you can choose your own branch/tag through the `ASDF_RUBY_BUILD_VERSION` variable:
 
 ```
 ASDF_RUBY_BUILD_VERSION=master asdf install ruby 2.6.4
@@ -63,6 +65,8 @@ like [rvm](https://github.com/rvm/rvm) or [chruby](https://github.com/postmodern
 note that you might have to change `.ruby-version` to include full version (e.g. change `2.6` to `2.6.1`).
 
 ## Troubleshooting
+
+> **_NOTE:_**  The most common issue reported for this plugin is a missing Ruby version. If you are not seeing a recent Ruby version in the list of available Ruby versions it's likely due to having an older version of this plugin. Run `asdf plugin-update ruby` to get the most recent list of Ruby versions.
 
 If you are moving to asdf-ruby from another Ruby version manager, it is recommended to completely uninstall the old Ruby version manager before installing asdf-ruby.
 

